src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -231,8 +229,11 @@ public AgentSettings LoadSettings()
                                                 args,
                                                 enforceSupplied);
 
-                // TODO: Scan for capabilites
-                var capabilities = new Dictionary<string, string>();
+                Dictionary<string, string> capabilities;
+                using (var ct = new CancellationTokenSource())
```

Comment:
Cancellation in config stage is done with CTRL-C, which terminates the process with "Environment.Exit". The cancellation token is used only when running the agent and calling from CreateSession.